user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_Create_CCAP_CM_pair.md

```diff
@@ -107,6 +107,9 @@ The elements will be created with the following specifications:
 > [!NOTE]
 > It may take some time before the elements are created. The larger the cluster, the longer it will take to create the elements.
 
+> > [!NOTE]
+> If a CISCO CBR-8 element was created, OFDM information will not be polled by and the Service Group names will use the Interface Alias by default. To change this, navigate to the newly created element(s) and update the Extra Configurations as needed.
```

Comment:
Something seems to be missing here: "will not be polled by ..." => Will not be polled by what? Or should the "by" be removed?
Also, is "Extra Configurations" a title of a page of those elements? Or if not, what does this refer to?

Reply:
Indeed, the by shouldn't be there, I put it there by mistake. And the Extra Configurations are on the Configuration visual of the element

Reply:
Could you confirm if the text looks OK the way I've adjusted it now?

Reply:
Yup, looks good